modules/proc_configuring-annotation-based-tracking-in-multiple-argo-cd-instances.adoc

```diff
@@ -0,0 +1,146 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-annotation-based-tracking-in-multiple-argo-cd-instances_{context}"]
+= Configuring annotation-based tracking in multiple Argo CD instances
+
+You can configure annotation-based tracking in multiple Argo CD instances.
+
+[NOTE]
+====
+This procedure uses the following example values:
+
+* `repoURL: https://github.com/redhat-developer/gitops-operator`
+* `server: https://kubernetes.default.svc`
```

Comment:
slight format change for an example URL:
```suggestion
* `repoURL`: `\https://github.com/redhat-developer/gitops-operator`
* `server`: `\https://kubernetes.default.svc`
```

Reply:
Fixed.